CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [v18.3.1] 2024-07-08
 
+### Fixed
+
+- **CUMULUS-3326**
```

Comment:
Should this be in the 18.3.1 section? Somewhat related: why was it added to 18.2.0 section on master?

Reply:
No, thanks for catching that 😦

Reply:
a62e3a36265